compiler/rec.vim

```diff
@@ -0,0 +1,15 @@
+" Vim Compiler File
+" Compiler:	rec
+
+if exists("current_compiler")
+    finish
```

Comment:
@T3SQ8 just to keep a consistent indentation in the file, could you use 2 spaces instead of 4 here?